master/trials.tx.up.sql

```diff
@@ -0,0 +1,76 @@
+-- keeping out of the migration directory for now to avoid having it run
```

Comment:
going to try to get rid of the aggs in the view, then do a proper migrations file